src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -138,7 +138,7 @@ protected void init(DerValue encoding, int req_type)
                         " req type is " + req_type);
 
                 System.out.println(">>> KDCRep: Message in bytes is =>");
-                byte[] dataBytes = encoding.getDataBytes();
+                byte[] dataBytes = encoding.getData().toByteArray();
```

Comment:
The javadoc for `getData()` says:
> Both this method and the data field should be avoided.
> Consider using data() instead.

Perhaps this should be:
```java
                byte[] dataBytes = encoding.data().toByteArray();
```

Reply:
The rest of this class is using the `getData()`, so I stayed with it for uniformity reasons. The `data()` is, also, not available in earlier versions of Java (like 11), and creates a new `DerInputStream` for no reason.

I can switch it to `data()`, if you feel we really have to, though.

Reply:
If there's no choice but to use `getData()` in earlier versions, that's something we'll just have to accept.

Using `getData()` is appropriate when the intent is to consume parts of the `DerValue` which is not the case here. Please use the recommended method.

Reply:
Done.